msdsn/conn_str.go

```diff
@@ -72,6 +74,15 @@ type Config struct {
 	ConnTimeout time.Duration // Use context for timeouts.
 	KeepAlive   time.Duration // Leave at default.
 	PacketSize  uint16
+
+	// Kerberos authentication fields
+
+	Krb5ConfFile      string
+	KrbCache          string
+	Realm             string
+	Initkrbwithkeytab string
+	Keytabfile        string
+	EnableKerberos    string
```

Comment:
If present, this should be a bool, but this might be better used with some type of enum.

Reply:
This is a flag which will decide if Kerberos will be used for authentication